examples/Serving models/Host GPT-2 (12-layer, 768-hidden, 12-heads, 117M parameters) .ipynb

```diff
@@ -0,0 +1,178 @@
+{
```

Comment:
Can you add instructions about how to install pytorch_transformers and details about the GPT-2 version being used?

Reply:
Done! (b017e19)